generated/thrift/rpc.thrift

```diff
@@ -140,7 +152,7 @@ struct Datapoint {
 struct FetchRawBatchRequest {
 	1: required i64 rangeStart
 	2: required i64 rangeEnd
-	3: required list<string> ids
+	3: required list<IDWithNamespace> idsWithNamespace
```

Comment:
For both fetch requests I think making them top level makes sense, we really don't need to fetch across namespaces during fetch requests.  It will also reduce the payload by up to 20% too if we do this.